File: go/go-leetcode/examples/concatenation_of_array.go
package main

import "fmt"

//https://leetcode.com/problems/concatenation-of-array/
//https://leetcode.com/submissions/detail/814888760/

func main() {
	fmt.Println(getConcatenation([]int{1, 2, 3}))
}

func getConcatenation(nums []int) []int {
	return append(nums, nums...)
}

File: python/python-leetcode/concatenation_of_array.py
from typing import List


# https://leetcode.com/problems/concatenation-of-array/
# https://leetcode.com/submissions/detail/814894266/

class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        return list(nums + nums)


sol = Solution()
print(sol.getConcatenation([1, 2, 3, 4, 4, 4]))
